AppleStore/AppleStore/WebViewController.swift

```diff
@@ -0,0 +1,135 @@
+        let requestURL = URLRequest(url: convertingURL)
+        contentWebView.load(requestURL)
+    }
+
+    private func setToolBar() {
+        let toolBar = UIToolbar(frame: CGRect(x: 0, y: view.bounds.height - 108, width: view.bounds.width, height: 50))
+        let backButton = UIBarButtonItem(image: UIImage(systemName: "arrow.backward"), style: .plain, target: self, action: #selector(goBack))
+        forwardButton = UIBarButtonItem(image: UIImage(systemName: "arrow.right"), style: .plain, target: self, action: #selector(goForward))
+        let refreshButton = UIBarButtonItem(image: UIImage(systemName: "arrow.clockwise"), style: .plain, target: self, action: #selector(refreshPage))
+        let shareButton = UIBarButtonItem(image: UIImage(systemName: "square.and.arrow.up"), style: .plain, target: self, action: #selector(shareActivity))
+        toolBar.barStyle = .black
+        toolBar.items = [backButton, forwardButton, UIBarButtonItem.flexibleSpace(), refreshButton, shareButton]
+        contentWebView.addSubview(toolBar)
+    }
+
+    private func setActivityIndicator() {
+        createActivityView()
+        createActivityIndicatorView()
+    }
+
+    private func showActivityIndicator(show: Bool) {
+        if show {
+            activityIndicatorView.startAnimating()
+        } else {
+            activityIndicatorView.stopAnimating()
+            activityView.removeFromSuperview()
+        }
+    }
+
+    private func createActivityView() {
+        activityView.frame = CGRect(x: 0, y: 0, width: 80, height: 80)
+        activityView.center = contentWebView.center
+        activityView.backgroundColor = UIColor.black
+        activityView.alpha = 0.3
+        activityView.layer.cornerRadius = 10
+        contentWebView.addSubview(activityView)
+    }
+
+    private func createActivityIndicatorView() {
+        activityIndicatorView.style = UIActivityIndicatorView.Style.large
+        activityIndicatorView.hidesWhenStopped = true
+        activityIndicatorView.center = contentWebView.center
+        contentWebView.addSubview(activityIndicatorView)
+    }
+
+    // MARK: - Actions
+    @objc private func goBack() {
+        if contentWebView.canGoBack {
+            contentWebView.goBack()
+        } else {
+            self.dismiss(animated: true, completion: nil)
+        }
+    }
+
+    @objc private func goForward() {
+        if contentWebView.canGoForward {
+            contentWebView.goForward()
+        }
+    }
+
+    @objc private func refreshPage() {
+        contentWebView.reload()
+    }
+
+    @objc private func shareActivity() {
+        let activity = UIActivityViewController(activityItems: [goodsURL], applicationActivities: nil)
+        present(activity, animated: true)
+    }
+}
+
+// MARK: - Extension: WKNavigationDelegate
+extension WebViewController: WKNavigationDelegate {
+    func webView(_ webView: WKWebView, didStartProvisionalNavigation navigation: WKNavigation?) {
+        if webView.canGoForward {
```

Comment:
```suggestion
       forwardButton.isEnabled = webView.canGoForward
```
не проще в 1 строку сделать? вместо 

if webView.canGoForward {
forwardButton.isEnabled = true
         } else {
             forwardButton.isEnabled = false
         }